yarr@jtoberling/files/yarr@jtoberling/desklet.js

```diff
@@ -87,7 +98,32 @@ class YarrDesklet extends Desklet.Desklet {
             return '255, 255, 255';
         return '0, 0, 0';
     }
+    
+    onAIPromptExample1() {
+        this.ai_systemprompt = 'Summarize in four sentences.';
+    }
+    
+    onAIPromptExample2() {
+        this.ai_systemprompt = 'Foglald össze limerick-ben.';
+    }
+    
+    onAIPromptExample3() {
+        this.ai_systemprompt = '俳句のエッセンス\n日本語では';
+    }
+    
+    onAIPromptExample4() {
+        this.ai_systemprompt = 'Foglald össze 4 mondatban.';
+    }
+    
+    onAIPromptExample5() {
+        this.ai_systemprompt = 'Summarize in 10 short lines.';
+    }
+    
```

Comment:
You can make the example prompts in English and add translation support to the strings. This will make the examples more useful. Shout if you need a hand with this.